public/css/list/host-service-list.less

```diff
@@ -0,0 +1,11 @@
+.controls .host-list .list-item,
+.controls .service-list .list-item {
+  .visual {
+    width: auto;
+
+    .state-ball {
+      // Align optically to `quick-actions`
+      margin-left: -1px;
```

Comment:
Again, feels arbitrary. Apply this to the visual.

Reply:
If it's only optically misaligned, I'd opt for leaving it out. Especially since we're talking about a single pixel. Otherwise it's even more arbitrary.

Reply:
I don't agree on the "only". 😉

But I'm fine with leaving it out.

Reply:
✅